README.md

```diff
@@ -14,9 +14,7 @@
 kubemci is a tool to configure Kubernetes ingress to load balance traffic across
 multiple Kubernetes clusters.
 
-This is a work in progress, not production ready yet.
-
-This is not an official Google product.
+This is a Google Cloud Platform [alpha](https://cloud.google.com/terms/launch-stages) product, not ready to be used in production yet.
```

Comment:
should we say anything in regards to v0.2? like "version 0.2 is our alpha release" or "With the release of v 0.2, we are now in alpha"?

Reply:
We decided not to tie alpha with 0.2.
The current HEAD version is good for alpha.

Reply:
okay, that's fine.
/lgtm